src/st/no3/44C64.c

```diff
@@ -255,16 +255,20 @@ u16 GetNormalizedAngle(u16 arg0, u16 arg1, u16 arg2) {
 }
 
 void SetStep(u8 step) {
```

Comment:
Do the psp versions of these not match everywhere?

Reply:
Yes, the PSP version should be the default. Fixed.

Reply:
It's still not the psp version?